src/core/public/chrome/ui/header/collapsible_nav.scss

```diff
@@ -24,3 +24,9 @@
     text-decoration: underline; /* 2 */
   }
 }
+
+@media (max-width: 1439px) {
```

Comment:
Not sure, but EUI may have constants and/or mixins around the most common break points? an hardcoded value probably should be avoided.